doc/build.rst

```diff
@@ -20,39 +16,60 @@ This page gives instructions on how to build and install Secure XGBoost from scr
 
    git clone --recursive https://github.com/mc2-project/mc2-xgboost.git
 
-Please refer to `Trouble Shooting`_ section first if you have any problem
+Please refer to the `Troubleshooting`_ section first if you have any problem
 during installation. If the instructions do not work for you, please feel free
-to ask questions at `the user forum <https://discuss.xgboost.ai>`_.
+to open an issue on `GitHub <https://github.com/mc2-project/mc2-xgboost/issues>`_.
 
 **Contents**
 
 * `Installing the Open Enclave SDK`_
 
 * `Installing Secure XGBoost Dependencies`_
 
-* `Building the Shared Library`_
+* `Building Secure XGBoost`_
 
-  - `Building on Ubuntu`_
+  - `Building the Targets`_
+  -  `Python Package Installation`_
```

Comment:
Could you delete the extra space here? There seem to be two spaces after the `-`

Reply:
Fixed!